2025 army changes

IRBosman opened the pull request:
2025 army changes

#53

IRBosman committed:
2025 army changes

2025 army changes

IRBosman committed:
Kislev fix

MattBlissett commented on public/json/warmaster-revolution/beastmen.json:
```diff
@@ -171,15 +171,15 @@
-        "If the ambush order is failed the ambushing group is not deployed and cannot ambush that turn, it can attempt to ambush in a subsequent turn at the same or a different place and at the same and different composition of units (distance, enemy within 20cm etc.) apply.",
+        "After deployment, Ambushers may receive further orders if in Command range, but the Ambush counts as their first order. If the order fails, Ambushers cannot deploy that turn but can try again later.",
         "",
-        "Ungor have a shooting range of 15cm."
```
This line shouldn't have been removed.

MattBlissett commented on public/json/warmaster-revolution/skaven.json:
```diff
@@ -141,6 +145,15 @@
+  "upgrades": {
+    "Rat Ogre Bodyguard": {
+      "order": 0,
+      "type": "Monstrous Mount",
```
I think this should be Special Mount.

MattBlissett reviewed (changes requested):
The "big guys" and "giant goes wild" changes are missing, see page 2 of the changes document. They should be applied to Albion, Bretonnia, Dogs of War, Goblins, Lizardmen, Norse, Ogre Kingdoms, Orcs, and you've already done it for Skaven.

MattBlissett commented:
Additionally, for completeness the WMR_Armies2.25.pdf file should replace the 2024 version in public/pdfs, and the link in src/views/Home.vue be updated.

dsusco commented:
Thanks, I'll try and take a look this week. This was everything in Ales's change doc?

IRBosman commented:
Creating a new commit with @MattBlissett's comments as we speak.

IRBosman committed:
Fixes 2025 changes WMR

- Re-add ungor line
- Update armies pdf to 2.25
- Big guys rule: decided to add this to the generic special rules and link it. This way we can easily link it in other stats should be be needed. Genericized wording.

IRBosman commented:
> "giant goes wild"

This is already in there via the generic Giant rule that's linked in all giant stat blocks (unless I missed one)
